service/awsconfig/v5/resource/cloudformation/resource_test.go

```diff
@@ -29,16 +29,33 @@ func Test_Resource_Cloudformation_GetCloudFormationTags(t *testing.T) {
 					Key:   aws.String("kubernetes.io/cluster/5xchu"),
 					Value: aws.String("owned"),
 				},
+				&awscloudformation.Tag{
+					Key:   aws.String("KubernetesCluster"),
+					Value: aws.String("5xchu"),
+				},
 			},
 		},
 	}
 
+	noContainsTag := func(tag *awscloudformation.Tag, tags []*awscloudformation.Tag) bool {
+		for _, inTag := range tags {
+			fmt.Printf("%t %t \n", tag, inTag)
+			if tag == inTag {
+				return true
+			}
+		}
+
+		return false
+	}
+
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
 			tags := getCloudFormationTags(tc.obj)
 
-			if !reflect.DeepEqual(tc.expectedTags, tags) {
-				t.Fatalf("Expected cloud formation tags %v but was %v", tc.expectedTags, tags)
+			for _, tag := range tc.expectedTags {
+				if noContainsTag(tag, tags) {
```

Comment:
For me naming the function containsTag and using `!containsTag` would be clearer.

I'd also prefer the function to be defined separately rather than in a closure.

Reply:
+1 for `containsTag` as suggested by Ross. I still prefer the closure, the function is only needed in this test and we don't need to bloat the top-level namespace, if it is required later somewhere else we can always refactor IMO.

Reply:
That makes sense. I'm fine with keeping the closure :)

Reply:
ok it was my first approach indeed hehe